README.md

```diff
@@ -73,6 +73,26 @@ Suppose we want to add a Kubic image. A suitable Vagrant box for such an image i
 
 By the way, more info on how to use Kubic images with vagrant-ceph is available in the [openSUSE:Ceph Wiki](https://en.opensuse.org/openSUSE:Ceph#Using_Rook_in_Vagrant_cluster).
 
+Following are two more examples for adding Vagrant boxes for openSUSE Leap 15.1 and openSUSE Tumbleweed, from the openSUSE download server:
+
+* __openSUSE Leap 15.1__
+
+`vagrant box add --provider libvirt --name virt-appl/openSUSE-Leap-15.1 https://download.opensuse.org/repositories/Virtualization:/Appliances:/Images:/openSUSE-Leap-15.1/images/Leap-15.1.x86_64-libvirt.box`
+
+* __openSUSE Tumbleweed__
```

Comment:
I think we need it only for 15.1, as Tumbleweed already automatically uploads from Virt to cloud.